omnigibson/robots/franka_allegro.py

```diff
@@ -228,3 +236,124 @@ def disabled_collision_pairs(self):
         return [
             ["link_12_0", "part_studio_link"],
         ]
+    
+    @property
+    def assisted_grasp_start_points(self):
+        return {self.default_arm: [
+            GraspingPoint(link_name=f"base_link", position=[0.015, 0, -0.03]),
+            GraspingPoint(link_name=f"base_link", position=[0.015, 0, -0.08]),
+            GraspingPoint(link_name=f"link_15_0_tip", position=[0, 0.015, 0.007]),
+        ]}
+
+    @property
+    def assisted_grasp_end_points(self):
+        return {self.default_arm: [
+            GraspingPoint(link_name=f"link_3_0_tip", position=[0.012, 0, 0.007]),
+            GraspingPoint(link_name=f"link_7_0_tip", position=[0.012, 0, 0.007]),
+            GraspingPoint(link_name=f"link_11_0_tip", position=[0.012, 0, 0.007]),
+        ]}
+
+    @property
+    def vr_rotation_offset(self):
+        return {self.default_arm: T.euler2quat(np.array([0, np.pi / 2, 0]))}
+    
+    def remap_thumb_to_allegro(self, coord: np.ndarray) -> np.ndarray:
+        """
+        remap VR thumb tracking data to allegro thumb, based on the bound of the two embodiments
+        Args:
+            bound (np.ndarray) 3D coord of thumb tracking position in the base_link frame
+        """
+        hand_bound = np.array([[0.005, 0.095], [-0.026, 0.104], [-0.005, 0.053]])    # bound for hand
```

Comment:
is this the link-aligned bbox?

Reply:
This is actually partially adapted from [Holo-Dex](https://holo-dex.github.io/). I'll add a reference in the comment